host/dynamicconfig.go

```diff
@@ -151,7 +145,7 @@ func (d *dynamicClient) GetDurationValue(
 		}
 	}
 	d.RUnlock()
-	return d.client.GetDurationValue(name, filters, defaultValue)
+	return d.client.GetDurationValue(name, filters)
 }
 
 func (d *dynamicClient) UpdateValue(name dynamicconfig.Key, value interface{}) error {
```

Comment:
separately: 
```go
func newIntegrationConfigClient(client dynamicconfig.Client) *dynamicClient {
```
ick.  can we move that to the test that uses it?  (as with others, 100% optional)